app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
we don't need this line until PATCH/POST requests, can you take it out to keep your code strictly TDD